spec/fabricators/question_fabricator.rb

```diff
@@ -1,2 +1,4 @@
 Fabricator(:question) do
+  content { Faker::Lorem.words(2).join(' ') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.